workers/unity/Packages/io.improbable.gdk.mobile/Editor/iOSLaunchUtils.cs

```diff
@@ -181,15 +181,22 @@ public static void Launch(DeviceLaunchConfig deviceLaunchConfig, MobileLaunchCon
 
         private static bool TryBuildXCodeProject(string developmentTeamId)
         {
-            return RedirectedProcess.Command("xcodebuild")
+            var result = RedirectedProcess.Command("xcodebuild")
                 .WithArgs("build-for-testing",
                     "-project", Path.Combine(XCodeProjectPath, XCodeProjectFile),
                     "-derivedDataPath", DerivedDataPath,
                     "-scheme", "Unity-iPhone",
                     $"DEVELOPMENT_TEAM={developmentTeamId}",
                     "-allowProvisioningUpdates")
-                .Run()
-                .ExitCode == 0;
+                .Run();
+
+            if (result.ExitCode == 0)
+            {
+                return true;
+            }
+
+            Debug.LogError(string.Join("\n", result.Stderr));
```

Comment:
```suggestion
            Debug.LogError($"Unable to build XCode project. {string.Join("\n", result.Stderr)}");
```

To be consistent with the other one?

Reply:
I initially thought this - but we catch and log this information in the caller

Reply:
perhaps we return IEnumerable<string> instead of bool and move this log there?

Reply:
In that case, shall we unify how we handle failures for these calls so its consistent? I don't mind either provided its similar